apps/core/scenes/bookingDetail/sectorDetails/segments/SegmentMapMarker.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react';
+import { MapView } from '@kiwicom/margarita-map';
+import { View } from 'react-native';
+import { StyleSheet, Text } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+type Props = {|
+  +latitude: ?number,
+  +longitude: ?number,
+  +name: ?string,
+|};
+
+export default function SegmentMapMarker(props: Props) {
+  if (props.latitude == null || props.longitude == null) {
+    return null;
+  }
+  return (
+    <MapView.Marker
+      coordinate={{
+        latitude: props.latitude,
+        longitude: props.longitude,
+      }}
+    >
+      <View style={styles.row}>
+        <View style={styles.dot} />
+        <Text size="small">{props.name}</Text>
+      </View>
+    </MapView.Marker>
+  );
+}
+
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: 'row',
+    position: 'absolute',
+    top: -9,
+    start: -3,
+    alignItems: 'center',
+  },
+  dot: {
+    backgroundColor: defaultTokens.paletteInkDark,
+    height: 6,
+    width: 6,
+    borderRadius: 3,
```

Comment:
We could use here `defaultTokens.borderRadiusNormal`, what do you think?

Reply:
If someone updates `defaultTokens.borderRadiusNormal` to be another value, this dot would no longer appear circular. They are by coincidence the same value.

Reply:
Oh, I didn't realise it should be a circle. But still there is token for it: `defaultTokens.borderRadiusCircle`

Reply:
![image](https://user-images.githubusercontent.com/7705041/53420832-d1fb3380-39dc-11e9-8275-ac1b1bb5fe81.png)